<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountingLSBillItemGUI</class>
 <widget class="QWidget" name="AccountingLSBillItemGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dati voce di computo con misure</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dataTab">
      <attribute name="title">
       <string>Dati</string>
      </attribute>
      <layout class="QGridLayout" name="dataTabLayout">
       <item row="1" column="0">
        <widget class="QGroupBox" name="priceGroupBox">
         <property name="title">
          <string>Prezzo</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="priceDataLayout">
          <item row="2" column="1">
           <widget class="QLineEdit" name="priceUnitMeasureLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="priceCodeLabel">
            <property name="text">
             <string>Codice</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="priceUnitMeasureLabel">
            <property name="text">
             <string>Unità di misura</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="priceShortDescLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="priceCodeLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="priceShortDescLabel">
            <property name="text">
             <string>Descrizione</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="PPUTotalLabel">
            <property name="text">
             <string>Costo Unitario</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="PPUTotalLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="amountsCheckBox">
         <property name="text">
          <string>Importo</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="valueDataGroupBox">
         <property name="title">
          <string>Misure</string>
         </property>
         <layout class="QGridLayout" name="valueDataLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="quantityLabel">
            <property name="text">
             <string>Quantità</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="quantityLineEdit">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QGroupBox" name="billItemLinesGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Dettaglio</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QPushButton" name="addBillItemLinePushButton">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="importBillItemMeasuresTXTPushButton">
               <property name="text">
                <string>TXT</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="3">
              <widget class="QTableView" name="billItemLinesTableView"/>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="delBillItemLinePushButton">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="valueDataVertSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="associateLinesCheckBox">
            <property name="text">
             <string>Associa misure</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="amountsGroupBox">
         <property name="title">
          <string>I&amp;mporto</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="amountsDataLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="totalAmountLabel">
            <property name="text">
             <string>Importo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="totalAmountLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="priceCheckBox">
         <property name="text">
          <string>Prezzo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="priceTab">
      <attribute name="title">
       <string>Prezzo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5"/>
     </widget>
     <widget class="QWidget" name="attributeTab">
      <attribute name="title">
       <string>Etichette</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QPushButton" name="addAttributePushButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="removeAttributePushButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableView" name="attributeTableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>amountsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>amountsCheckBox</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>amountsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>amountsGroupBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>amountsGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>amountsGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>amountsGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>amountsCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priceCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>priceCheckBox</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priceCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>priceGroupBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priceGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>priceGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priceGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>priceCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
